packages/itwinui-css/src/tile/tile.scss

```diff
@@ -175,9 +196,31 @@
     border-bottom: 1px solid t(iui-color-background-border);
   }
 
+  &.iui-disabled {
+    filter: grayscale(100%);
+    cursor: not-allowed;
+    border-bottom: 4px solid t(iui-color-background-border);
+  }
+
   .iui-thumbnail-icon {
     @include iui-icons-3xl;
   }
+
+  &.iui-negative {
```

Comment:
These also might be not needed...
Instead, you could have a private variable (i.e. `_iui-tile-thumbnail-border-bottom`) and set its different values on root

```
...
&.iui-negative {
  _iui-tile-thumbnail-border-bottom: 4px solid var(--iui-icons-color-negative);
}
```

So you would move up the default value from line 196 and update it per different status class

Reply:
Fixed

Reply:
I dont think you did...
Specificity is higher with current solution. Variables could help solve that.

Reply:
Okay, now I understand what you are saying, let me try to implement it